service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -105,24 +114,57 @@ class TestRunSetsApiController {
   @Autowired private ObjectMapper objectMapper;
 
   @Test
-  void chainCallsTogether() throws Exception {
```

Comment:
There's probably a case to be made for a super simple version of this test (chain calls together) as well as all of these different "should continue even with 1 failed run" variations. Maybe they can all call some "run a runset" base case, with special behavior expectations per test case?

Reply:
I am not sure how to actually have a "run a runset" base case without putting it in each test 🤔 Should I refactor the mock code out of each tests?